app/frontend/src/components/UserSummary.tsx

```diff
@@ -8,34 +8,28 @@ import { Skeleton } from "@material-ui/lab";
 import React from "react";
 
 import { COMMUNITY_STANDING } from "../features/constants";
-import { hostingStatusLabels } from "../features/profile/constants";
 import { User } from "../pb/api_pb";
 import Avatar from "./Avatar";
 import ScoreBar from "./Bar/ScoreBar";
-import { CouchIcon } from "./Icons";
-import IconText from "./IconText";
 
-const useStyles = makeStyles((theme) => ({
-  root: { display: "flex", alignItems: "center", padding: 0 },
+export const useStyles = makeStyles((theme) => ({
+  root: { display: "flex", alignItems: "center", padding: 0, width: "100%" },
```

Comment:
nit: formatting? put a comma after the `width: "100%"` + alphabetize

Reply:
Surprised Prettier didn't auto format this and left it...